tests/test_collections.py

```diff
@@ -111,11 +125,19 @@ def test_patch(self):
         contents = [col_file_item] * 8 + [col_ptr_item] * 8
         uuid, version = self._put(contents)
 
-        for patch_payload in [dict(),
-                              dict(description="foo", name="cn"),
-                              dict(description="bar", details={1: 2}),
-                              dict(add_contents=contents),
-                              dict(remove_contents=contents)]:
+        expected_contents = {'contents': [col_file_item,
+                                          col_ptr_item],
+                             'description': 'd',
+                             'details': {},
+                             'name': 'n',
+                             'owner': 'travis-test@human-cell-atlas-travis-test.iam.gserviceaccount.com'
```

Comment:
This should not be hardcoded here. I think it's OK if the test would just not assume anything about the owner.